content/wordpress/index.md

```diff
@@ -26,5 +26,7 @@ Please go through the following chapters.
 
 * [PHPUnit](//wordpress/phpunit.md)
 
+* [Templating using wp.template](/wordpress/wp-template.md)
```

Comment:
Create a `Javascript Reference` page and in that page add details about `wp.template`. So `Javascript Reference` page will also contain any WP js related details in future.

Reply:
@chandrapatel I have addressed the required feedbacks